workos/src/main/kotlin/com/workos/portal/PortalApi.kt

```diff
@@ -50,14 +76,20 @@ class PortalApi(val workos: WorkOS) {
       }
     }
 
+    /**
+     * @suppress
+     */
     companion object {
       @JvmStatic
-      fun builder(): Builder {
-        return Builder()
+      fun builder(): GeneratePortalLinkOptionsBuilder {
+        return GeneratePortalLinkOptionsBuilder()
       }
     }
   }
 
+  /**
+   * Generate an Admin Portal Link.   *
```

Comment:
Looks like an extra `*`.

```suggestion
   * Generate an Admin Portal Link.
```